ds/ds.php

```diff
@@ -1636,9 +1795,9 @@ public function sum()
          * Creates a new map using values from the current instance and another
          * map.
          *
-         * A ∪ B = {x: x ∈ A ∨ x ∈ B}
+         * A ∪ B = {<b>x:</b> x ∈ A ∨ x ∈ B}
```

Comment:
What is the purpose of selection of `x:`? Wasn't it made by mistake?

Reply:
Formatting was by mistake. I don't see how that could have happened, since the original docs don't have these marks. I removed it.

Reply:
By a regex (similar to `\w+:`), I included markings for "Notice:" and "Caution:" in the doc blocks and that must have got over "x:" too.